### DISCUSSION TITLE
Change Background Image of Login Page

### DISCUSSION THREAD

**Discussion by BaroDevelopment** [requarks/wiki#4240] [Help / Questions]
Hello,
is it possible to change the background image of /login ?
![image](https://user-images.githubusercontent.com/8499987/126882302-623814b7-add5-47bf-810e-fdc8189b4e7b.png)

**Comment by cinatic** [comment]
Settings -> Security -> Background Image

**Reply by BaroDevelopment** [reply]
Thanks
